scripts/Muon/GUI/ElementalAnalysis2/grouping_widget/ea_grouping_table_widget_presenter.py

```diff
@@ -121,15 +122,16 @@ def handle_rebin_option_column_changed(self, col, changed_item, workspace_name):
             params = changed_item.text().split(":")
             if len(params) == 2:
                 if params[0] == "Steps":
-                    self._model.handle_rebin(name=workspace_name, rebin_type="Fixed", rebin_param=float(params[1]))
+                    self._model.handle_rebin(name=workspace_name, rebin_type="Fixed",
+                                             rebin_param=float(params[1][0:-3]))
```

Comment:
why does this have `[0:-3]`?

Reply:
this removes the units "KeV" at the end of string.

Reply:
could we have a comment explaining that?